src/tools/crawler/main.rs

```diff
@@ -81,8 +82,8 @@ async fn main() {
         });
     }
 
-    // Wait for the connection to be complete.
-    wait_until!(Duration::from_secs(3), crawler.node().num_connected() >= 1);
+    // Wait for one of the seed nodes to respond with a list of addrs.
+    wait_until!(Duration::from_secs(120), crawler.known_network.nodes().len() > args.seed_addrs.len());
```

Comment:
since we can potentially need to wait for a long time, it's probably best that we alter the check interval for `wait_until` between `#[cfg(test)]` (the current value) and `#[cfg(not(test))]` (probably sth like 200ms)